packages/framework/src/Resources/views/Admin/Layout/displayUuid.html.twig

```diff
@@ -0,0 +1,6 @@
+{% set keys = _context|keys %}
+{% if (keys[1] is defined) and (attribute(_context[keys[1]], 'uuid') is defined) %}
```

Comment:
`key[1]` assumes an entity passed to the template as the first variable? Or why 1?